src/controllers/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
```

Comment:
i would love to see you start coding using ECMA6

Reply:
What should we do here?